spec/models/async_transaction/base_spec.rb

```diff
@@ -57,7 +57,7 @@
   end
 
   describe '.stale scope' do
-    it 'finds "old" transactions but not new ones' do
+    it 'finds "old" transactions but not new ones', run_at: 'Tue, 12 Dec 2017 03:09:06 GMT' do
```

Comment:
As a member of the team that implemented this service, I feel partially responsible for this :) 

This spec was intended to test whether or not a transaction is eligible to be destroyed, based on the `DELETE_COMPLETED_AFTER` const in the base class.

What do you think about replacing the spec with something like this one:

```ruby
    it 'finds transactions that are eligible to be destroyed' do
      stale_age  = AsyncTransaction::Base::DELETE_COMPLETED_AFTER + 1.day
      active_age = AsyncTransaction::Base::DELETE_COMPLETED_AFTER - 1.day

      stale_transaction = create(
        :address_transaction,
        created_at: (Time.current - stale_age).iso8601,
        status: AsyncTransaction::Base::COMPLETED
      )
      create(
        :telephone_transaction,
        created_at: (Time.current - active_age).iso8601,
        status: AsyncTransaction::Base::COMPLETED
      )

      transactions = AsyncTransaction::Base.stale

      expect(transactions.count).to eq 1
      expect(transactions.first).to eq stale_transaction
    end
```

That should be more expressive of the intent of the what is under test, and mitigate any special calendar days since the const is using the `month` method, which Rails is smart about.

Reply:
sounds good to me, i replaced the spec